sdk/identity/azure-identity/src/main/java/com/azure/identity/DefaultAzureCredentialBuilder.java

```diff
@@ -220,11 +221,12 @@ public DefaultAzureCredentialBuilder additionallyAllowedTenants(List<String> add
     /**
      * Specifies a {@link Duration} timeout for developer credentials (such as Azure CLI) that rely on separate process
      * invocations.
-     * @param processTimeout The {@link Duration} to wait.
+     * @param developerCredentialTimout The {@link Duration} to wait.
      * @return An updated instance of this builder with the timeout specified.
      */
-    public DefaultAzureCredentialBuilder processTimeout(Duration processTimeout) {
-        this.identityClientOptions.setProcessTimeout(processTimeout);
+    public DefaultAzureCredentialBuilder developerCredentialTimeout(Duration developerCredentialTimout) {
```

Comment:
In changelog it says developerProcessTimeout, which one is correct ?

Reply:
The change which team is going with is, 'credentialProcessTimeout'

Reply:
done